components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -2478,24 +2491,31 @@ public long getAPISubscriptionCountByAPI(APIIdentifier identifier)
                                                                                       AppManagementException {
 
         Map<String, List> users = new HashMap<String, List>();
-        // List<WebAppInfoDTO> webAppInfoDTOList;
+        String sqlQuery;
         List<Subscriber> subscribers;
-
-        String sqlQuery =
-                "SELECT " + "SUBR.USER_ID AS USER_ID,  API.APP_NAME AS API,API.APP_VERSION, API.APP_PROVIDER AS PROVIDER,SUB.SUBSCRIPTION_TIME as TIME  "
-                        + " FROM APM_SUBSCRIBER SUBR, APM_APPLICATION APP, APM_SUBSCRIPTION SUB, APM_APP API "
-                        + " WHERE SUB.APPLICATION_ID = APP.APPLICATION_ID AND SUBR.SUBSCRIBER_ID = APP.SUBSCRIBER_ID AND"
-                        + " SUB.APP_ID = API.APP_ID AND SUBR.TENANT_ID = ? AND SUB.SUBSCRIPTION_TIME BETWEEN ? AND ? " 
-                        + " GROUP BY SUBR.USER_ID,API.APP_NAME,API.APP_VERSION";
-
-
         Connection connection = null;
         PreparedStatement ps = null;
         ResultSet result = null;
 
         try {
             connection = APIMgtDBUtil.getConnection();
-
+            sqlQuery = "SELECT SUBR.USER_ID AS USER_ID,API.APP_NAME AS API, "
+                    + "API.APP_VERSION, API.APP_PROVIDER AS PROVIDER, "
+                    + "SUB.SUBSCRIPTION_TIME as TIME  FROM APM_SUBSCRIBER SUBR, "
+                    + "APM_APPLICATION APP, APM_SUBSCRIPTION SUB, APM_APP API "
+                    + "WHERE SUB.APPLICATION_ID = APP.APPLICATION_ID AND "
+                    + "SUBR.SUBSCRIBER_ID = APP.SUBSCRIBER_ID AND "
+                    + "SUB.APP_ID = API.APP_ID AND SUBR.TENANT_ID = ? AND "
+                    + "SUB.SUBSCRIPTION_TIME BETWEEN ";
+
+            if (!connection.getMetaData().getDriverName().contains("Oracle")) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/97624962445a2a9c12448bd5e61bfbba719bf8f3